position_offset:
  - 0.0  # was 11 
  - 0.0 # was 17
  - 0.0  

orientation_offset:
  - 0.0  
  - 0.0  
  - 0.0
  - 0.0
